easybuild/easyconfigs/c/CUDA/CUDA-9.0.176.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'EB_CUDA'
```

Comment:
@smoors As I mentioned, this is not needed, EasyBuild will use `EB_CUDA` by default based on the software name if no `easyblock` is specified...

Reply:
oops sorry, I thought I deleted it..